linchpin/provision/roles/azure/tasks/provisionVM.yml

```diff
@@ -39,7 +42,7 @@
   poll: 0
   register: res_def_output
   when: _async
```

Comment:
Should add this back?  not debug_mode

Reply:
you are right, I was lazy and forgot about it.